### ISSUE TITLE
While disabling an instance that is already disabled

### ISSUE THREAD

**Issue by netrules** [richtr/NoSleep.js#105]
https://github.com/richtr/NoSleep.js/blob/master/src/index.js#L105

* Device Name: w/e
* Device Version: doesn't matter
* Browser Name: Chrome
* Browser Version: latest?
* NoSleep.js Version: cdnjs version

### Expected Behavior
Sorry for the informality of the configuration -- it is a simple fix. Add check for `this._wakeLock` to see if instance is currently running before calling `release()` method.

### Actual Behavior
![image](https://user-images.githubusercontent.com/48886164/89851304-2b680b80-db5a-11ea-9bd0-6b5a108afaf5.png)
An instance of NoSleep, called `_nosleep`, using `_nosleep.disable()` once it has already been called, crashes the rest of my expected script behaviour.

### Code
https://github.com/richtr/NoSleep.js/blob/master/src/index.js#L105
Change
```js
      this._wakeLock.release();
```

to
```js
      if(this._wakeLock)
            this._wakeLock.release();
```

I expect this to be a solution but I haven't tested it yet. I will use state management in my code meanwhile.

### How Do We Reproduce?
Use NoSleep.disable() twice

**Comment by Tharit** [comment]
The same error is thrown if the wakelock could not be acquired when enabling.
this._wakeLock is only set when the wakelock is acquired successfully, otherwise it says null. The caller does not know about this failure, so HAS to call disable, which then fails.

The suggested fix above would solve this issue as well!

**Comment by richtr** [comment]
Fixed by 7aa6c05928d3be452444d569b6fa4422d08deb50!
